cmd/helm/installer/install.go

```diff
@@ -141,6 +142,7 @@ func generateDeployment(opts *Options) *v1beta1.Deployment {
 							},
 							Env: []v1.EnvVar{
 								{Name: "TILLER_NAMESPACE", Value: opts.Namespace},
+								{Name: "TILLER_HISTORY_MAX", Value: fmt.Sprintf("%d", opts.MaxHistory)},
```

Comment:
The former goes through about five layers of parsing and delegation to land at [`fmt.(*fmt).fmt_integer`](https://sourcegraph.com/github.com/golang/go@2673f9ed23348c634f6331ee589d489e4d9c7a9b/-/blob/src/fmt/format.go#L193-195)`.

By contrast, `strconv.Itoa` goes through two calls to land at either [`strconv.small`](https://sourcegraph.com/github.com/golang/go@2673f9ed23348c634f6331ee589d489e4d9c7a9b/-/blob/src/strconv/itoa.go#L56-64) or [`strcovn.formatBits`](https://sourcegraph.com/github.com/golang/go@2673f9ed23348c634f6331ee589d489e4d9c7a9b/-/blob/src/strconv/itoa.go#L90-97).

The latter has fewer features to accommodate and bottoms out faster, including its optimization for small numbers in with a radix of 10.